synctl/cli.py

```diff
@@ -384,11 +384,19 @@ def __init__(self) -> None:
             "memLimit": 3000,
             "imageSize": 1500,
         }
+        self.ism = {
+            "testCount": 2000,
+            "frequency": 1,
+            "cpuLimit": 1000,
+            "memLimit": 300,
+            "imageSize": 350,
+        }
 
         self.factors = {
             "browserTest": 1,
             "APIScript": 0.042,
-            "APISimple": 0.025
+            "APISimple": 0.025,
+            "SSLTest": 0.025
```

Comment:
let us use "ISMTest" since we will also support other test types in ISM playback engine, they use the same pricing factor